app/lib/till.rb

```diff
@@ -0,0 +1,47 @@
+require_relative 'order'
+require_relative 'receipt'
+require_relative 'payment'
+require_relative 'menu'
+
+
+class Till
+
+  include Menu
+
+  attr_reader :order, :receipt, :tax, :change
+
+  def initialize
+    @tax = 8.64
+    menu_read_price_list
+    new_order
+  end
+
+  def new_order
+    @order = Order.new
+    @receipt = nil
```

Comment:
Do we really need to store receipt as a variable? Does this happen in a real till or does it just generate it at the end?